src/components/BottomSheetInLineNotification.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
```

Comment:
it's not really clear to me how this component will evolve, so i've made it quite specific to inline notifications rather than very generic to save time